applications/EigenSolversApplication/README.md

```diff
@@ -126,23 +126,22 @@ In case you have installed [MKL](https://software.intel.com/en-us/mkl), you can
 
 ## Install MKL on Ubuntu with apt
 
-Open a terminal window and run the following commands to install MKL from the official Intel repository:
+Open a terminal window and run the following commands to install MKL from the official Intel repository.
+For more details see https://software.intel.com/en-us/articles/installing-intel-free-libs-and-python-apt-repo.
 
 ```bash
-apt-get update -y
-apt-get upgrade -y
-apt-get install -y gnupg2 software-properties-common wget
```

Comment:
i think those are not necessary (except wget in the next line) but i don't think we need to add that in the instruction